antora-docs/modules/ROOT/pages/authoring.adoc

```diff
@@ -0,0 +1,37 @@
+= Policy Authoring
```

Comment:
Were there any changes to the content of this file other than formatting?

Reply:
The only change was to change the Markdown headings to Asciidoc style headings